policy.md

```diff
@@ -226,24 +317,205 @@ Each audit statement MUST be accompanied by documentation of the audit team qual
 * How the audit team and team members are bound by law, regulation or professional standards to render an independent assessment of the CA (e.g., https://pub.aicpa.org/codeofconduct/Ethics.aspx# 0.300.050 Objectivity and Independence; CPA Canada, Rule 204; or Annex A of ETSI EN 319 403/403-1, respectively); and
 * Whether the audit team relied on any third-party specialists or affiliate audit firms, and if so, their names and where they performed services.
 
-### 5.3 Annual CCADB Self-Assessments ###
+## 6. Additional Required Practices
+
+### 6.1 Disclosing and Responding to Incidents
+
+The CCADB [Incident Reporting Guidelines (IRGs)](https://www.ccadb.org/cas/incident-report) intend to present a common format for publicly disclosing incidents. CA Owners who are required to publicly disclose incidents MUST adhere to the CCADB IRGs. Those CA Owners MUST update the incidents and respond to questions in accordance with the IRGs until closure.
+
+### 6.2 Certificate Revocation List Disclosures
+
+For each unexpired and unrevoked CA certificate disclosed to the CCADB and within 7 days of the corresponding CA issuing its first certificate, CA Owners MUST disclose either:
+- the URL of a full and complete Certificate Revocation List (CRL); or
+- a JSON Array of Partitioned CRL URLs.
+
+URLs:
+- MUST match exactly as they appear in the certificates issued by the corresponding CA.
+
+If populating a full and complete CRL URL: 
+- the corresponding CRL SHOULD NOT contain an 'Issuing Distribution Point' extension.
+- the JSON Array of Partitioned CRL URLs field MUST be empty.
+
+If populating a JSON Array of Partitioned CRL URLs: 
+- CA Owners MUST ensure that each corresponding CRL contains a critical 'Issuing Distribution Point' extension and the 'distributionPoint' field of the extension MUST include a 'UniformResourceIdentifier'. The value of the UniformResourceIdentifier MUST exactly match a URL, from which the CRL was accessed, present in the CCADB record associated with the CA certificate.
+- the full and complete CRL URL MUST be empty.
+
+Under normal operating conditions, the CRL URLs provided by CAs in this section MUST be available such that relying parties are able to successfully retrieve the current CRL every 4 hours.
```

Comment:
"in **_accordance with_** this section"

Reply:
Addressed in https://github.com/ryancdickson/www.ccadb.org/commit/52c7b33fdd3fe256ff3cde154b48fb18f37df843.